src/tests/data/misc/roughtime_response.vec

```diff
@@ -0,0 +1,68 @@
+[Botan::Invalid_Argument:tag not found]
```

Comment:
Don't hardcode exception messages into the test cases. Instead just have two blocks, valid and invalid, and (optionally) for invalid cases, comments explaining why they are invalid.